code/mob/living/carbon/human.dm

```diff
@@ -3026,8 +3035,17 @@
 		// added log - cirr
 		logTheThing("combat", src, src.chest_item, "takes damage from farting out [src.chest_item] embedded in [src]'s chest cavity at [log_loc(src)]")
 		// Make copy of item on ground
+#if ASS_JAM
+		if (istype(src.chest_item,/mob/))
+			var/mob/outChestMob = src.chest_item
+			outChestMob.set_loc(get_turf(src))
+		else
+			var/obj/item/outChestItem = src.chest_item
+			outChestItem.set_loc(get_turf(src))
+#else
```

Comment:
Is this even needed?

Reply:
The code or the feature? Because the code is there so I don't have to change the nonassjam code to allow ejecting mobs, since it casts to item. As for the feature, its to allow ejecting people without surgery/gibbing, for stuff like hiding an accomplice inside you for crime purposes.

Reply:
the code, because at a glance it looks like it would still work without explicitly casting to mob

Reply:
Yeah just tested it, I'm dumb. Must have mistaken a bug during testing as being caused by not casting to mob.